docs/deployment/k8s/index.mdx

```diff
@@ -60,29 +64,49 @@ config:
 helm upgrade --install -f values.yaml nats-cluster nats/nats
 ```
 
-#### Running wadm
+#### Deploying wadm
 
-You can deploy wadm to your Kubernetes cluster with the wadm Helm chart. First, clone the wadm repo from GitHub:
+You can deploy wadm to your Kubernetes cluster with a Helm chart hosted by wasmCloud. Use this `values.yaml` file to define a minimal deployment that connects with the NATS server deployed above: 
 
-```shell
-git clone https://github.com/wasmCloud/wadm.git
+```yaml
+wadm:
+  config:
+    nats:
+      server: "nats-cluster.default.svc.cluster.local:4222"
 ```
 
-In the cloned repo, find the chart at `wadm/charts/wadm`. In your `values.yaml` file, the `server` field for NATS should be set to:
+Run `helm install`:
 
-```yaml
-server: 'nats-cluster.default.svc.cluster.local:4222'
+```shell
+helm install wadm -f values.yaml --version 0.2.2 oci://ghcr.io/wasmcloud/charts/wadm

```

Comment:
In the interest that we improve the wadm chart in the future, I might recommend we drop `--version 0.2.2` to have the end-user pick up the latest version, wdyt?
```suggestion
helm install wadm -f values.yaml oci://ghcr.io/wasmcloud/charts/wadm
```